cli/src/template/android/app/src/main/java/com/example/moproapp/MainActivity.kt

```diff
@@ -1,4 +1,4 @@
-package com.example.moproapp
+package com.mopro.mopro_app
```

Comment:
I think you should not change the package name

Reply:
I got the error
```sh
e: file:///Users/zhengyawen/Documents/GitHub/mopro-example-app-noir/android/app/src/main/java/com/example/moproapp/FibonacciComponent.kt:33:19 Unresolved reference: getFilePathFromAssets
```
because it is not in the same package

Reply:
fixed in [fc8bd82](https://github.com/zkmopro/mopro/pull/399/commits/fc8bd822cdf3b5c8cc5be9abac87752e89e6e4f5)